Core/include/Acts/TrackFitting/GlobalChiSquareFitter.hpp

```diff
@@ -541,27 +611,15 @@ class Gx2Fitter {
       // TODO generalize for non-2D measurements
       for (size_t iMeas = 0; iMeas < gx2fResult.collectorResiduals.size();
            iMeas++) {
-        ActsMatrix<2, eBoundSize> proj;
-
-        for (size_t i_ = 0; i_ < 2; i_++) {
-          for (size_t j_ = 0; j_ < eBoundSize; j_++) {
-            proj(i_, j_) = 0;
-          }
-        }
-        proj(0, 0) = 1;
-        proj(1, 1) = 1;
-
         const auto ri = gx2fResult.collectorResiduals[iMeas];
```

Comment:
In a way bad for this nominclature that we have chosen CamelCase, it would look so math-obvious if you wrote `r_i `, but then other cases would probably be different.